share/spack/gitlab/cloud_pipelines/stacks/aws-isc-aarch64/spack.yaml

```diff
@@ -0,0 +1,245 @@
+        mpi:
+          - openmpi
+          - mpich
+      variants: +mpi
+    binutils:
+      variants: +ld +gold +headers +libiberty ~nls
+      version:
+        - 2.36.1
+    doxygen:
+      version:
+        - 1.8.20
+    elfutils:
+      variants: +bzip2 ~nls +xz
+    hdf5:
+      variants: +fortran +hl +shared
+    libfabric:
+      variants: fabrics=efa,tcp,udp,sockets,verbs,shm,mrail,rxd,rxm
+    libunwind:
+      variants: +pic +xz
+    mesa:
+      variants: ~llvm
+    mesa18:
+      variants: ~llvm
+    mpich:
+      variants: ~wrapperrpath netmod=ofi device=ch4
+    ncurses:
+      variants: +termlib
+    openblas:
+      variants: threads=openmp
+    openmpi:
+      variants: fabrics=ofi +legacylaunchers
+    openturns:
+      version: [1.18]
+    relion:
+      variants: ~mklfft
+#    texlive:
+#      version: [20210325]
+    trilinos:
+      variants: +amesos +amesos2 +anasazi +aztec +belos +boost +epetra +epetraext +ifpack +ifpack2 +intrepid +intrepid2 +isorropia +kokkos +ml +minitensor +muelu +nox +piro +phalanx +rol +rythmos +sacado +stk +shards +shylu +stokhos +stratimikos +teko +tempus +tpetra +trilinoscouplings +zoltan +zoltan2 +superlu-dist gotype=long_long
+    xz:
+      variants: +pic
+
+  definitions:
+
+  - compiler_specs:
+    - gcc@11.2
+    # Licensing OK?
+    # - intel-oneapi-compilers@2022.1
+    # - nvhpc
+
+  - cuda_specs:
+    # Depends on ctffind which embeds fsincos (x86-specific asm) within code. Will not build on ARM
+    #- relion +cuda cuda_arch=70
+    - raja +cuda cuda_arch=70
+    - mfem +cuda cuda_arch=70
+
+  - app_specs: 
+    - bwa
+# Depends on simde which requires newer compiler?
+    #- bowtie2
+    # Requires x86_64 specific ASM
+    #- cistem
+    - cromwell
+    - fastqc
+    - flux-sched
+    - flux-core
+    - flux-pmix
+    - gatk
+    - gromacs
+    - lammps
+    - wrf build_type=dm+sm
+    - mfem
+    - mpas-model ^parallelio+pnetcdf
+    - nextflow
+    - octave
+    - openfoam
+    - osu-micro-benchmarks
+    - parallel
+    - paraview
+    - picard
+    - quantum-espresso
+    - raja
+# Depends on bowtie2 -> simde which requires newer compiler?
+    #- rsem
+    # Errors on texlive
+    #- rstudio
+    - salmon
+    - samtools
+    - seqtk
+    - snakemake
+    - star
+    # Requires gcc@9:
+    #- ufs-weather-model
+# requires LLVM which fails without constraint
+    #- visit
+    
+  - lib_specs:
+    - openmpi fabrics=ofi
+    - openmpi fabrics=ofi +legacylaunchers
+    - openmpi fabrics=auto
+    - mpich
+    - libfabric
+
+  - compiler:
+    - '%gcc'
+
+  - target: 
+    - 'target=aarch64'
+    - 'target=graviton2'
+
+
+  specs:
+
+  - matrix:
+    - - $cuda_specs
+    - - $compiler
+    - - $target
+
+  - matrix:
+    - - $app_specs
+    - - $compiler
+    - - $target
+
+  - matrix:
+    - - $lib_specs
+    - - $compiler
+    - - $target
+
+  - matrix:
+    - - $compiler_specs
+    - - $compiler
+    - - $target
+
+
+  mirrors: { "mirror": "s3://spack-binaries/aws-e4s" }
```

Comment:
I think it makes sense to change this to something resembling the stack name.  Maybe:

```
s3://spack-binaries/aws-isc-aarch64
```

Reply:
Ok, can change this. Was hoping to have all packages merged into the central buildcache. Used this aws-e4s to merge the builds from this pipeline and https://github.com/spack/spack/pull/29522.